gc/base/standard/Scavenger.cpp

```diff
@@ -4594,9 +4596,11 @@ MM_Scavenger::internalPreCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *sub
  * Perform any post-collection work as requested by the garbage collection invoker.
  */
 void
-MM_Scavenger::internalPostCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace)
+MM_Scavenger::internalPostCollect(MM_EnvironmentBase *envBase, MM_MemorySubSpace *subSpace)
 {
-	calcGCStats((MM_EnvironmentStandard*)env);
+	MM_EnvironmentStandard *env = (MM_EnvironmentStandard *)envBase;
```

Comment:
We do have more formal way to cast:
`	MM_EnvironmentStandard *env = MM_EnvironmentStandard::getEnvironment(envBase);`